README.md

```diff
@@ -484,14 +484,16 @@ Similar to the JaCoCo `dump` command the flag `--coverage_dump=<file>` specifies
 `coverage.exec`, that should be generated after the fuzzing run. It contains a binary representation of the gathered
 coverage data in the JaCoCo format.
 
-The JaCoCo `report` command can be used to generate reports based on the coverage dump. For example the
-following command generates an HTML report in the folder `./report/` containing all classes available in `classes.jar`
-and their coverage as captured in the export `coverage.exec`.
+The JaCoCo `report` command can be used to generate reports based on the coverage dump. The internally used version
+differs slightly from the official one, and to prevent incorrect results it should be preferred.
+For example the following command generates an HTML report in the folder `/tmp/report/` containing all classes available in
+`/classes.jar` and their coverage as captured in the export `/coverage.exec`.
 ```shell
-java -jar jacococli.jar report coverage.exec \
-  --classfiles classes.jar \
-  --sourcefiles some/path/to/sources \
-  --html ./report/ \
+bazel run @jazzer_jacoco//:jacoco_cli -- \
```

Comment:
I like that we offer a way to build this in-repo. Maybe we could just also add a link to the fork's release page to the preceding paragraph and provide instructions for both, with the fork's instructions coming first?

Reply:
Good idea, updated the readme.